includes/snmp.inc.php

```diff
@@ -264,6 +282,29 @@ function snmp_get($device, $oid, $options = null, $mib = null, $mibdir = null)
 }//end snmp_get()
 
 
+function snmp_get_next($device, $oid, $options = null, $mib = null, $mibdir = null)
```

Comment:
No preference to not use get_next it's just we haven't had the need for it before so we want to make sure we don't introduce functions that aren't needed.

I'm assuming you can't just bulkwalk?